wormhole-connect/src/views/Bridge/RouteOptions.tsx

```diff
@@ -281,6 +283,11 @@ function RouteOption(props: { route: RouteData; disabled: boolean }) {
     [route, toChain],
   );
 
+  const displayName = isNttRoute(routeName)
+    ? getNttDisplayName(fromTokenConfig, toTokenConfig) || props.route.name
```

Comment:
if this wasnt one-off hack only on the release branch I would cringe at putting it here but we good